.github/ISSUE_TEMPLATE/1-bug_report.yaml

```
@@ -0,0 +1,204 @@
+
+        *Example:*
+        > Windows 10 Version 1607 Build 14393.1066
+    validations:
+      required: true
+
+  - type: textarea
+    id: otherVersions
+    attributes:
+      label: Name and version of other software in use when reproducing the issue
+      description: |
+        Typically, you can get this information from the "About" dialog of the software.
+
+        *Example:*
+        > Chrome version: 67.0.3396.87
+
+  - type: textarea
+    id: otherSysInfo
+    attributes:
+      label: Other information about your system
+      description: |
+        This is other information about your setup that you think might be relevant to us.
+
+        *Example:*
+        > Running Windows 10 in a VM
+
+  - type: markdown
+    attributes:
+      value: |
+        ### Troubleshooting steps
+
+        The following questions ask about the basic investigation steps you may have completed.
+        Please feel free to add more information here to tell us about what you have tried.
+
+  - type: input
+    id: restart
+    validations:
+      required: true
+    attributes:
+      label: Does the issue still occur after restarting your computer?
+      description: |
+        Restarting your computer will help to ensure that software is in a fresh state.
+
+  - type: input
+    id: addons
+    validations:
+      required: true
+    attributes:
+      label: If NVDA add-ons are disabled, is your problem still occurring?
+      description: |
+        In many cases issues could be caused by an add-on not working correctly or conflicting with other features already integrated in NVDA itself.
+        Therefore, it is recommended to reproduce issues with all add-ons disabled.
+        You can restart NVDA with add-ons disabled through the exit dialog.
+        In case the issue is caused by an add-on, it is recommended to contact the author of the add-on first.
+
+  - type: input
+    id: comRegFix
+    validations:
+      required: true
+    attributes:
+      label: Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
+      description: |
+        COM DLL files, which the IAccessible COM interface depends on, can get unregistered after installing and uninstalling different programs on a computer.
+        The consequence is that NVDA in certain cases does not work properly.
+        Specifically, it reports "unknown" when trying to navigate websites or fails to switch between focus and browse mode, it causes performance issues, focus instability and other odd problems.
+        The COM registry fixing tool has been introduced in order to re-register those DLL files.
+        In any case, re-registering those DLL files does not have any negative impact on a computer's functionality.
+        Thus, it is recommended to run the fixing tool whenever focus problems, performance problems on websites or navigation problems in focus or browse mode on different interfaces are encountered.
```

I think it makes sense to mention that running the tool also happens during installation. There's no need to rerun the tool just after an update or install. It is also an extra point to prove that running the tool won't impose any harm.
```suggestion
        Thus, it is recommended to run the fixing tool whenever focus problems, performance problems on websites or navigation problems in focus or browse mode on different interfaces are encountered.
        Note that running the tool is part of NVDA's installation process.
```